pipeline/plot/unit_characteristic_plot.py

```diff
@@ -708,3 +799,18 @@ def plot_paired_coding_direction(unit_g1, unit_g2, labels=None, time_period=None
 
 def get_m_scale(shank_count):
     return 1350 - 150*shank_count
+
+
+def _get_ccf_xyz_max():
+    xmax, ymax, zmax = ccf.CCFLabel.aggr(ccf.CCF, xmax='max(ccf_x)', ymax='max(ccf_y)', zmax='max(ccf_z)').fetch1(
+        'xmax', 'ymax', 'zmax')
+    locals()['_ccf_xyz_max'] = xmax, ymax, zmax
+    return xmax, ymax, zmax
+
+
+def _get_ccf_vox_res():
+    voxel_res = (ccf.CCFLabel & 'ccf_label_id = 0').fetch1('ccf_resolution')
```

Comment:
the in-code attributes on CCFLabel table definition can be used here if desired to save a query/fetch

Reply:
^ keeping fetch